packages/ui-react/src/hooks/eas/use-eas-schema-form.ts

```diff
@@ -0,0 +1,113 @@
+import { gql } from "@geist/graphql";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useQuery } from "@tanstack/react-query";
+import { rawRequest } from "graphql-request";
+import { useMemo } from "react";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { getEasscanEndpoint } from "#lib/eas/easscan.js";
+
+type SchemaByQuery = any;
+
+// function to get JSON from attestation (we cannot dynamically create zod schemas)
+export const getZodSchemaFromSchemaString = (schemaString: string) => {
+	const stringPairs = schemaString.split(",");
+	const entries = stringPairs.map((pair) => pair.split(" "));
+	const zodSchemaObject: any = {};
+
+	for (const entry of entries) {
```

Comment:
EAS parse the schema (of ABI) 
you can refer to implementation at SchemaEncoder constructor to which prase the schmeaString to schema object (then see if we are able to create zod schema from it)